packages/utilities/fast-design/package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Will this be public or private?  We'll want this value for Beachball if we want to release npm packages.